test/native/helpers.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { act, render, fireEvent } from '@testing-library/react-native';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	subscribeParentGetHtml,
+	provideToNative_Html as provideToNativeHtml,
+} from '@wordpress/react-native-bridge';
+// Editor component is not exposed in the pacakge because is meant to be consumed
+// internally, however we require it for rendering the editor in integration tests,
+// for this reason it's imported with path access.
+// eslint-disable-next-line no-restricted-syntax
+import Editor from '@wordpress/edit-post/src/editor';
```

Comment:
Following with [this comment](https://github.com/WordPress/gutenberg/pull/32038#discussion_r644972782), the Editor component is now included using `import` instead of `require`.